dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -244,6 +335,16 @@ private void addImagePath(final int weight,
           }
           pathBuilder.append(weight > 0? "/resize_w/" + weight : StringPool.BLANK);
           pathBuilder.append(height > 0? "/resize_h/" + height : StringPool.BLANK);
+          if(focalPoint.isPresent()) {
+              pathBuilder.append("/fp/" + focalPoint.get());
+          }
+          if(cropWidth>0) {
+              pathBuilder.append("/crop_w/" + cropWidth);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid concatenating nonliterals in a StringBuffer/StringBuilder constructor or append().](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)